backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -498,6 +501,7 @@ trait StandardAsyncExecutionActor
           |touch $stdoutRedirection $stderrRedirection
           |tee $stdoutRedirection < "$$$out" &
           |tee $stderrRedirection < "$$$err" >&2 &
+          |tail -q -f $stdoutRedirection $stderrRedirection > $taskLogRedirection &
```

Comment:
NB this change was really meant only for the GCP Batch backend but here has been made in code that is common for the PAPI v2 backend